get-involved.html

```diff
@@ -34,54 +34,68 @@
     <h2 id="contribute" class="title section-title">Contribute</h2>
     <div class="columns is-centered">
       <div class="column">
-        <h3 id="programmers" class="is-size-3">Programmers</h3>
+        <h3 id="everyone" class="is-size-3">Everyone</h3>
         <div class="content">
           <ul>
             <li>
-              Work on a <a href="https://dev.minetest.net/Modding_Intro">game or mod</a>,
-              and publish it on <a href="https://content.minetest.net">ContentDB</a>
-              or <a href="https://forum.minetest.net">the forums</a>.
+              <a href="#reporting-issues">Report bugs and request features</a>.
             </li>
             <li>
-              Report bugs and submit patches on
-              <a href="https://github.com/minetest">GitHub</a> or <a href="https://wiki.minetest.net/IRC">IRC</a>.
+              Give support on the <a href="https://forum.minetest.net">forums</a> and <a href="https://wiki.minetest.net/IRC">IRC</a>.
+            </li>
+            <li>
+              Help translating Minetest using
```

Comment:
Maybe this sounds better? Not sure.

```suggestion
              Help translate Minetest using
```